packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -80,6 +82,16 @@ class ExpansionTile extends StatefulWidget {
   /// Specifies if the list tile is initially expanded (true) or collapsed (false, the default).
   final bool initiallyExpanded;
 
+  /// The tween to control the color of the header.
+  /// begin defaults to theme.textTheme.subtitle1.color
+  /// end defaults to theme.accentColor
+  final ColorTween headerColorTween;
+
+  /// The tween to control the color of the header.
+  /// begin defaults to theme.unselectedWidgetColor
+  /// end defaults to theme.accentColor
```

Comment:
/// The tween that controls the icon's color.
  ///
  /// This tween's `begin` property defaults to [ThemeData.unselectedWidgetColor],
  /// whereas the tween's `end` property defaults to [ThemeData.accentColor].